frontend/styles/Navbar.module.css

```diff
@@ -0,0 +1,426 @@
+}
+
+.homeGridItem {
+  grid-area: home;
+}
+
+.aboutUsGridItem {
+  grid-area: aboutUs;
+}
+
+.eventsGridItem {
+  grid-area: events;
+}
+
+.opportunitiesGridItem {
+  grid-area: opportunities;
+}
+
+.mediaGridItem {
+  grid-area: media;
+}
+
+.joinUsGridItem {
+  grid-area: joinUs;
+}
+
+.logoContainer {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  padding-top: 3%;
+  padding-left: 12%;
+}
+
+.linkContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  height: 100%;
+  width: 90%;
+  font-size: 16px;
+  font-family: "Montserrat", sans-serif;
+  text-transform: uppercase;
+}
+
+.linkContent {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  transition: color 0.3s;
+}
+
+.linkContent:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.linkContent:hover .linkUnderline {
+  opacity: 1;
+  width: 100%;
+}
+
+.linkUnderline {
+  position: absolute;
+  height: 90%;
+  width: 0%;
+  opacity: 0;
+  border-bottom: 0.15vw solid var(--red);
+  transition: width 0.25s, opacity 0.25s;
+}
+
+.currentPageUnderline {
+  width: 100%;
+  opacity: 1;
+}
+
+.dropdownContainer {
+  display: block;
+}
+
+.dropdownContainer:hover .linkContent {
+  color: var(--red) !important;
+}
+
+.dropdownContainer:hover .linkUnderline {
+  width: 100%;
+  opacity: 1;
+}
+
+.dropdownContainer:hover .dropdownContent {
+  visibility: visible;
+  opacity: 1;
+}
+
+.dropdownContainer:hover .clearDropdownContent {
+  visibility: visible;
+  opacity: 1;
+}
+
+.dropdownContent {
+  position: absolute;
+  visibility: hidden;
+  width: 15%;
+  background-color: var(--light-yellow);
+  opacity: 0;
+  border-radius: 0px 0px 15px 15px;
+  transition: background-color 0.2s, border-radius 0.2s, opacity 0.2s, visibility 0.2s;
+}
+
+.clearDropdownContent {
+  background-color: transparent !important;
+  border: 0.15vw solid var(--white);
+  border-top-right-radius: 15px;
+}
+
+.dropdownItem {
+  position: relative;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  padding: 5% 15% 0% 15%; /* no bottom padding */
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItem:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.dropdownItem:hover a {
+  color: var(--red);
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItemEnd {
+  position: relative;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  padding: 5% 15% 8% 15%;
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItemEnd:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.dropdownItemEnd:hover a{
+  color: var(--red);
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.currentPageBold {
+  font-weight: bold;
+}
+
+.icon {
+  margin-left: -10px;
+  margin-right: 10px;
+  transform: scale(0.9);
+}
+
+/* compact navbar */
```

Comment:
Follow the CSS commenting convention for bigger comments (take a look at `global.css` for reference)